### ISSUE TITLE
Latest Release label not updated

### ISSUE THREAD

**Issue by StefRe** [opacapp/opacclient#596]
The [github opac client front page](https://github.com/opacapp/opacclient) still shows release 5.8.0 as the latest:
![grafik](https://user-images.githubusercontent.com/19879328/96103012-76462c80-0ed7-11eb-97cd-231bc2c11da8.png)

**Comment by johan12345** [comment]
It seems that "Releases" are not created automatically for each tag. I just manually created a new release for the newest version. Not sure if we can automate this...
@raphaelm do you have an idea?

**Comment by raphaelm** [comment]
yep, apparently is possible with actions: https://github.com/marketplace/actions/automatic-releases
